bindings/nodejs/lib/types/block/core/block.ts

```diff
@@ -118,6 +119,13 @@ class Block {
     asValidation(): ValidationBlockBody {
         return this.body.asValidation();
     }
+
+    /**
+     * Gets the block id.
+     */
+    id(params: ProtocolParameters): BlockId {
+        return Utils.blockId(this, params);
```

Comment:
This should be added like here in lib.rs https://github.com/iotaledger/iota-sdk/blob/1539710218d4d8d71c8400da0d6257ef2074d027/bindings/nodejs/lib/index.ts#L19-L34
otherwise webpack will fail since it depends on the binary

Reply:
I thought we only did that because it was a breaking change to move it to a different module?

Reply:
To which module would you move it?

Reply:
We moved it to this place so it's higher up in the dependency structure, when importing just the type, this doesn't get loaded